frontend/src/app/general/header/header.component.html

```diff
@@ -21,85 +21,89 @@
               color="primary"
               [attr.href]="item.href"
               [attr.target]="item.target"
+              class="text-xs"
             >
               {{ item.name }}
               <mat-icon *ngIf="item.icon">{{ item.icon }}</mat-icon>
             </a>
-          </ng-container>
-          <ng-template #router>
-            <a mat-raised-button color="primary" [routerLink]="item.routerLink">
+          } @else {
+            <a
+              mat-raised-button
+              color="primary"
+              [routerLink]="item.routerLink"
+              class="text-xs"
+            >
               {{ item.name }}
             </a>
-          </ng-template>
-        </ng-container>
-      </div>
+          }
+        }
+      }
     </div>
-    <div>
-      <div class="hidden gap-2 xl:flex">
-        <div>
-          <a
-            mat-raised-button
-            color="primary"
-            href="https://github.com/DSD-DBS/capella-collab-manager/issues"
-            target="_blank"
-            >Open Issue on Github <mat-icon>open_in_new</mat-icon>
-          </a>
-        </div>
-        <mat-menu #profileMenu="matMenu">
-          <a
-            class="profileMenuButton"
-            mat-menu-item
-            [routerLink]="['user', userService.user?.id]"
-          >
-            Profile <mat-icon>account_circle</mat-icon>
-          </a>
-          <a
-            *ngIf="userService.user?.role === 'administrator'"
-            class="profileMenuButton"
-            mat-menu-item
-            routerLink="settings"
-          >
-            Settings <mat-icon>settings</mat-icon>
-          </a>
-          <a
-            *ngIf="userService.user?.role === 'administrator'"
-            class="profileMenuButton"
-            mat-menu-item
-            routerLink="events"
-          >
-            Events <mat-icon>event_note</mat-icon>
-          </a>
-          <a class="profileMenuButton" mat-menu-item routerLink="tokens">
-            Tokens <mat-icon>key</mat-icon>
-          </a>
-          <a
-            class="profileMenuButton"
-            mat-menu-item
-            routerLink="logout/redirect"
-            [queryParams]="{ reason: 'logout' }"
-            *ngIf="authService.isLoggedIn()"
-          >
-            Log out <mat-icon>logout</mat-icon>
-          </a>
-        </mat-menu>
+    <div class="hidden items-center xl:flex">
+      <a
+        mat-raised-button
+        color="primary"
+        href="https://github.com/DSD-DBS/capella-collab-manager/issues"
+        target="_blank"
+        class="ml-2 text-xs"
+        >Open Issue on Github <mat-icon>open_in_new</mat-icon>
+      </a>
+
+      <mat-menu #profileMenu="matMenu" class="flex !w-full items-center pr-3">
+        <a
+          class="profileMenuButton text-xs"
+          mat-menu-item
+          [routerLink]="['user', userService.user?.id]"
+        >
+          Profile <mat-icon>account_circle</mat-icon>
+        </a>
```

Comment:
Moritz is right, moving the right padding from the menu to the links enables the hover effect to cover the entire element. I removed some CSS that didn't seem to have any effect, and I think this overall looks more standard as a dropdown menu (icon followed by left aligned text).

```<mat-menu #profileMenu="matMenu" class="flex items-center">```

header.component.css:
```
.profileMenuButton {
  padding: 0 15px 0 15px;
  text-align: left;
}
```
<img width="159" alt="MenuDropdown" src="https://github.com/DSD-DBS/capella-collab-manager/assets/79602065/6ff206ad-7185-40e6-88e6-3d143da81bbb">

Reply:
(Though it might be nicer to get rid of the external CSS and just use `class="px-[15px] text-left text-xs"` on each link)

Reply:
thanks a lot!

> (Though it might be nicer to get rid of the external CSS and just use `class="px-[15px] text-left text-xs"` on each link)
mmh but then we will need to copy the code for every menu item don't we?

Reply:
alright then I will add it - thanks!